assets/js/autocomplete/display.js

```diff
@@ -16,7 +17,7 @@ function updateSuggestions (term) {
   const template = autocompleteResultsTemplate({
     empty: results.length === 0,
     results: results,
-    term: term
+    term: escapeHtmlEntities(term)
```

Comment:
With the current escaping I would remove this escape and use 

```
<div class="title">"<em>{{term}}</em>"</div>
```

in `autocomplete-suggestions.js` to escape the term

Reply:
This suggestion can be implemented.

Reply:
done